### PR TITLE
feat(wizard): embedded wizard

### LINKED ISSUE #841: Embedded Wizard: HTML/CSS is needed
The [design documentation for the Wizard](http://www.patternfly.org/pattern-library/communication/wizard/) shows an example of an embedded wizard.

This issue is to track the implementation of the html and css for the embedded wizard design.

### PR DESCRIPTION
**Pull request by matthewcarleton**
## Description
This is the embedded version of the wizard.

https://rawgit.com/matthewcarleton/patternfly/wizard-embedded-added-dist/dist/tests/wizard-embedded.html

## PR checklist (if relevant)

- [x] **Cross browser**: works in IE9
- [x] **Cross browser**: works in IE10
- [x] **Cross browser**: works in IE11
- [x] **Cross browser**: works in Edge
- [x] **Cross browser**: works in Chrome
- [x] **Cross browser**: works in Firefox
- [x] **Cross browser**: works in Safari
- [ ] **Cross browser**: works in Opera
- [x] **Responsive**: works in extra small, small, medium and large view ports.
- [ ] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

Fixes #841

### CODE REVIEW

**Review by jeff-phillips-18 — DISMISSED** [review]

File: src/less/wizard.less [hunk 1]
```diff
@@ -566,3 +566,36 @@
     }
   }
 }
+//
+//
+// This is for the embedded wizard.
+// --------------------------------------------------
+.wizard-embedded-pf {
+  &-main {
+    overflow: scroll;
```
Comment (andresgalante) [inline src/less/wizard.less]: this is generating scroll bars

File: src/less/wizard.less [hunk 2]
```diff
@@ -566,3 +566,36 @@
...
+    padding:0;
```
Comment (andresgalante) [inline src/less/wizard.less]: please add a space after the `:` we need a formatter!

**Review by andresgalante — CHANGES_REQUESTED** [review]
Here are scroll bars and a strange gap at the bottom of the wizard. Is that intended? screen shot 2018-01-15 at 10 53 19 am

File: src/less/wizard.less (hunk 2 above)
Comment (matthewcarleton) [reply src/less/wizard.less]: :)

File: src/less/wizard.less (hunk 1 above)
Comment (matthewcarleton) [reply src/less/wizard.less]: Are you referring to the padding on the body? That is on all the test pages.

Comment (andresgalante) [reply src/less/wizard.less]: I am referring to the scroll bars, they are always on, even though there is not content to scroll.

Comment (matthewcarleton) [reply src/less/wizard.less]: No I mean the gap comment

**Comment by matthewcarleton** [comment]
@andresgalante I think the scrolling issue is fixed now. I've added an extension for Chrome to show them and I don't see them now with overflow:auto. Can you confirm?

**Comment by LHinson** [comment]
Based on our meeting with Mary, sounds like this not a requirement at this time and no reason to implement given the limitations it has re: flexibility.

**Pull request closed without merging**